Gruntfile.js

```diff
@@ -201,6 +201,8 @@ module.exports = function (grunt) {
         src: ['<%= yeoman.app %>/index.html'],
         ignorePath:  /\.\.\//,
         exclude: [
+          // bower registration error? we get 2x versions of uri.js/urijs
```

Comment:
you had some uri.js changes in the NPM PR i just marked to merge, make sure these arent going to conflict